src/AvaloniaEdit/Search/SearchPanel.xaml

```diff
@@ -37,8 +37,17 @@
                 VerticalAlignment="Top" TextBlock.FontFamily="Segoi UI" TextBlock.FontSize="10">
                     <Grid ColumnDefinitions="Auto,Auto,Auto"
                 RowDefinitions="Auto,Auto">
+                        <Popup Name="PART_MessageView"
+                               Classes="ToolTip"
+                               PlacementMode="Bottom"
+                               StaysOpen="True"
+                               Focusable="False"
+                               IsOpen="False"
+                               ObeyScreenEdges="True">
+                            <ContentControl>Message Here</ContentControl>
+                            </Popup>
                         <ToggleButton Classes="ExpanderToggle" Name="Expander" VerticalAlignment="Top"
-                                      IsChecked="{TemplateBinding IsReplaceMode, Mode=TwoWay}"
+                                      IsChecked="{Binding IsReplaceMode, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
                                       Grid.Column="0"
```

Comment:
If you just needed a two way binding, Avalonia has Mode in TemplateBinding

Reply:
It was template binding previously, but it doesn't work.